### ISSUE TITLE
CSV import fails

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#7156]
### Observed behavior

Reported by @nucleogenesis in

https://www.notion.so/learningequality/CSV-Upload-Fails-logs-stuff-inside-50cab450cba04793b46ec13c9dde131f

Fails to upload with error:

```
{detail: "Multipart form parse error - Invalid boundary in multipart: None"}
```

Perhaps related cc @jredrejo: 

* https://github.com/learningequality/kolibri/issues/6750
* https://github.com/learningequality/kolibri/pull/6768

### Steps to reproduce

Try to import CSV on beta server

### Context

0.14.0 beta 3

**Comment by jredrejo** [comment]
It's a different error, after #6969 , content-type multipart is not working properly

**Comment by jredrejo** [comment]
Fixed after #7168 has been merged
